src/index.js

```diff
@@ -24,10 +24,24 @@ class LevelDatastore {
   /* :: db: levelup */
 
   constructor (path /* : string */, opts /* : ?LevelOptions */) {
-    this.db = levelup(path, Object.assign({}, opts, {
-      compression: false, // same default as go
-      valueEncoding: 'binary'
-    }))
+    // Default to leveldown db
+    const database = opts && opts.db ? opts.db : require('leveldown')
+    delete opts.db
```

Comment:
needs an existence check like the line above otherwise `ReferenceError: opts is not defined` for `null`/`undefined`

Reply:
...and a test? ;)